docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
+
+services:
+ postgres:
+   restart: always
+   image: postgres:latest
+   environment:
+     POSTGRES_USER: openoversight
+     POSTGRES_PASSWORD: terriblepassword
+     POSTGRES_DB: openoversight-dev
+
+ web:
+   restart: always
+   build:
+      context: .
+      dockerfile: ./dockerfiles/web/Dockerfile
+   volumes:
+     - ./OpenOversight/:/usr/src/app/OpenOversight/
+   links:
+     - postgres:postgres
+   expose:
+     - "3000"
+   command: /usr/local/bin/gunicorn -w 4 -b 0.0.0.0:3000 app:app
+   ports:
+     - "3000:3000"
+     - "5432:5432"
```

Comment:
As written, this binds the host's port 5432 to the `web` container's 5432, right? If I understand docker-compose syntax, to make the postgres server accessible to the host, this line would need to go up to a `ports` stanza on `postgres`, right? If we don't want to make it host-accessible, we can just remove this line. (You already updated `CONTRIB.md` to show executing psql within the container, so _theoretically_ we might just not want to expose it to the host at all by default?)